PWGHF/vertexingHF/AliAnalysisTaskHFSimpleVertices.cxx

```diff
@@ -1482,7 +1470,7 @@ Int_t AliAnalysisTaskHFSimpleVertices::SelectInvMassAndPt2prong(TObjArray* trkAr
   if(minv2>lolim*lolim && minv2<hilim*hilim ) retval += (1 << kbitDzero);
   pdg2[0]=321; pdg2[1]=211;   // K+ pi- --> D0bar
   minv2 = rd4massCalc2->InvMass2(nprongs,pdg2);
-  if(minv2>lolim*lolim && minv2<hilim*hilim && (retval & (1 << kbitDzero)) == 0) retval+=(1 << kbitDzero);
+  if(minv2>lolim*lolim && minv2<hilim*hilim && (retval & (1 << kbitDzerobar)) == 0) retval+=(1 << kbitDzerobar);
```

Comment:
we do not need to check ((retval (1 << kbitDzerobar)==0) here. It is sufficient to have:
if(minv2>lolim*lolim && minv2<hilim*hilim) retval+=(1 << kbitDzerobar);

Reply:
Ok, Thank you so much for your reminding.